nextcloudappstore/core/static/assets/js/ui/app-detail.js

```diff
@@ -41,6 +41,16 @@
             });
     }
 
+    function load_language(lang) {
+        commentTextarea.readOnly = true;
+        fetch(commentUrl + "?lang=" + lang, {credentials: 'include'})
+            .then((response) => response.json())
+            .then((json) => {
+                commentTextarea.value = json.comment;
+        });
+        commentTextarea.readOnly = false;
```

Comment:
This part will execute after you fired the request, not when it finished successfully, see https://github.com/github/fetch/issues/223#issuecomment-148927226